CRM/Core/CodeGen/Util/Smarty.php

```diff
@@ -48,28 +48,16 @@ public function getCompileDir() {
   public function createSmarty(): Smarty {
     $base = dirname(__DIR__, 4);
     $pkgs = file_exists(dirname($base) . "/civicrm-packages") ? dirname($base) . "/civicrm-packages" : "$base/packages";
-    require_once $pkgs . '/smarty4/vendor/autoload.php';
+    require_once $pkgs . '/smarty5/Smarty.php';
     $smarty = new Smarty();
     $smarty->setTemplateDir("$base/xml/templates");
     $pluginsDirectory = $smarty->getPluginsDir();
```

Comment:
@eileenmcnaughton doesn't this need to go as well?

Reply:
@seamuslee001 but how else would we set the variable that we don't use?